models/company_graph.py

```
@@ -0,0 +1,1968 @@
+
+
+# =============================================================================
+# WORKFLOW EXECUTION MODELS
+# =============================================================================
+
+
+class RepoScanResult(BaseModel):
+    """Result of a repository scan with detected stack and systems."""
+    model_config = {"extra": "ignore"}
+
+    scan_id: str = Field(default_factory=lambda: secrets.token_hex(8), description="Unique scan identifier")
```

Ruff (0.15.14)

[error] Undefined name `secrets`

(F821)